src/chip.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {createUseStyles} from './jss';
+import {useTheme} from './hooks';
+import Box from './box';
+import {Text2} from './text';
+import IconButton from './icon-button';
+import IconCloseRegular from './generated/mistica-icons/icon-close-regular';
+
+const useStyles = createUseStyles(({colors}) => ({
+    container: {
+        display: 'inline-flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        border: `1px solid ${colors.border}`,
+        borderRadius: 16,
+    },
+}));
+
+type ChipProps = {
+    value: string;
```

Comment:
I think I prefer children here. As we do in Tag

Reply:
changed